apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -71,10 +72,22 @@ class SectionsPage extends Component {
     this.props.loadClassroomList();
   };
 
-  handleImportClose = (selectedId) => {
+  handleImportCancel = () => {
     this.setState({rosterDialogOpen: false});
-    // TODO (josh): use `selectedId`.
-    console.log(selectedId);
+  };
+
+  handleImport = courseId => {
+    this.props.importClassroomStarted();
+
+    $.getJSON('/dashboardapi/import_google_classroom', { courseId }).then(() => {
+      this.setState({rosterDialogOpen: false});
+      this.setState({sectionsLoaded: false});
```

Comment:
These two can be combined into one line (i.e. you can set multiple state field in a single call to setState)